File: linguagem-java/desafios/fFatorial.java
/*Fatorial:
        Crie uma função que calcule o fatorial de um número inteiro positivo fornecido pelo usuário.*/

import java.util.Scanner;

public class fFatorial {
    public static void main(String[] args) {
        Scanner sc = new Scanner(System.in);
        System.out.println("Escreva um numero acima de zero e que não seja letra");
        int leitura = sc.nextInt();
        System.out.println("Você escreveu: " + leitura);
        for (int i = leitura-1; i >= 1; i--){
                leitura *= i;
                System.out.println(leitura);
        }

    }
}

File: linguagem-javascript/desafios/f-fatorial.py
#Fatorial:
#Crie uma função que calcule o fatorial de um número inteiro positivo fornecido pelo usuário.
leitura = (int(input("Digite um numero para ser fatorado: " )))
j = 1
k = leitura - (leitura-1)
i = leitura
while j < i:
    leitura = leitura*j
    j +=1
print("Esse numero em fatorial é:",leitura)
